exchange/index.go

```diff
@@ -194,6 +208,12 @@ func (idx *Index) loadFromStore() error {
 	return nil
 }
 
+func (idx *Index) UpdateOptions(opts ...IndexOption) {
```

Comment:
To test the GC in TestCommit, I had to add a way to change the bounds limit used for the eviction. Not sure if it's the most elegant way, I reused the same principle as with IndexOption